docs/content/how-to/aws/create-aws-hosted-cluster-spot-instances.md

```diff
@@ -0,0 +1,216 @@
+---
```

Comment:
I would expect this to just be a section of https://hypershift.pages.dev/how-to/automated-machine-management/ focusing on the nodepool peculiarities, no need to redocument cluster creation